### DISCUSSION TITLE
Just a post to say thank you

### DISCUSSION THREAD

**Discussion by CarlEdZa** [turkaysoft/glow#4] [General]
I don't have any comments or suggestions, yet, but I do want to thank you for producing a truly useful tool.
Much appreciated!

**Comment by turkaysoft** [comment]
> I don't have any comments or suggestions, yet, but I do want to thank you for producing a truly useful tool. Much appreciated!

Thank you for your valuable opinion and nice comment.

The purpose of Glow, unlike other alternative software, is to make it more modern, more comprehensive, more user-friendly, so that even someone who doesn't know anything about computers can easily get information about their system.

I will continue to improve it as much as my own experience and knowledge allow.

Best regards.

**Comment by CarlEdZa** [comment]
Thank you for all the years of hard work that you put into Glow.
It is a great product and that is due to your efforts!
You should be proud of what you created.
I wish you all of the best going forward.

**Reply by turkaysoft** [reply]
Thank you for your good wishes.
Glow has contributed a lot to me. It has become as good as other system analysis software.

It is time for me to sail to new seas.
